### ISSUE TITLE
Fix the build on `main` branch

### ISSUE THREAD

**Issue by 43081j** [es-tooling/eslint-plugin-jsx-a11y-x#12]
Currently the doc generator fails. This is likely because we still need to rename to `eslint-plugin-jsx-a11y-x` in various places.

The test workflow also fails last time I checked, but not sure why yet.

cc @JounQin
